phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexTool.java

```diff
@@ -429,10 +429,7 @@ private long getMaxRebuildAsyncDate(String schemaName, List<String> disableIndex
             
         }
 
-        private Job configureJobForAysncIndex()
-
-                throws Exception {
-
+        private Job configureJobForAysncIndex() throws Exception {
```

Comment:
nit: please fix spelling error in "async". (yes I know it was already there but it's been bugging me for months! :-) )